sdks/how-to/how-to-manage-healthcare-elements.md

```diff
@@ -4,3 +4,283 @@ description: Learn how to manage healthcare elements
+      version: '20020131',
+    }),
+  ]),
+  openingDate: new Date('2019-10-12').getTime(),
+})
+
+const healthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  newHE,
+  patient.id,
+)
+```
+
+:::note
+
+The value of some parameters, such as the id of the Healthcare Element, its author or the responsible Data Owner will be
+filled automatically by the backend.
+
+:::
+
+When creating a new Healthcare Element, you must specify the Patient it is associated to.  
+If the method runs successfully, the Promise will return the newly created Healthcare Element as it is on the database.
+It is also possible to create a series of Healthcare Elements that describe a medical history. In a medical history, 
+the healthcare elements share the same `healthcareElementId`
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create multiple related HEs as data owner-->
+```typescript
+const startHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'The patient has been diagnosed Pararibulitis',
+    codes: new Set([
+      new CodingReference({
+        id: 'SNOMEDCT|617|20020131',
+        type: 'SNOMEDCT',
+        code: '617',
+        version: '20020131',
+      }),
+    ]),
+    openingDate: new Date('2019-10-12').getTime(),
+  }),
+  patient.id,
+)
+
+const followUpHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'The patient recovered',
+    openingDate: new Date('2020-11-08').getTime(),
+    healthcareElementId: startHealthcareElement.healthcareElementId,
+  }),
+  patient.id,
+)
+```
+
+:::note
+
+The `healthcareElementId` is the id of the first Healthcare Element of the series.
+
+:::
+
+Several unrelated Healthcare Elements can also be created at once.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create multiple HEs as data owner-->
+```typescript
+const healthcareElement1 = new HealthcareElement({
+  description: 'The patient has been diagnosed Pararibulitis',
+  codes: new Set([
+    new CodingReference({
+      id: 'SNOMEDCT|617|20020131',
+      type: 'SNOMEDCT',
+      code: '617',
+      version: '20020131',
+    }),
+  ]),
+  openingDate: new Date('2019-10-12').getTime(),
+})
+
+const healthcareElement2 = new HealthcareElement({
+  description: 'The patient has also the flu',
+  openingDate: new Date('2020-11-08').getTime(),
+})
+
+const newElements = await api.healthcareElementApi.createOrModifyHealthcareElements(
+  [healthcareElement1, healthcareElement2],
+  patient.id,
+)
+```
+
+:::caution
+
+Even if you associate a Healthcare Element to a Patient, the Patient does not have access to it unless you give access 
+to them.
+
+:::
+
+
+
+## Sharing a Healthcare Element with a Patient
+
+After creating the Healthcare Element, the Healthcare Professional shares it with the Patient.
```

Comment:
After creating the Healthcare Element, the Healthcare Professional can share it with the Patient.

Plus, add more detail / a note to explain that a Patient can give access to a HCP, to another patient, a hcp can give access to a patient, to another hcp, ... it's not one way